front/app/resources/GetSerieFromStream.tsx

```diff
@@ -154,9 +150,13 @@ export default class GetSerieFromStream extends PureComponent<Props, State> {
   }
 
   render() {
-    const { children } = this.props;
+    const { children, convertToGraphFormat } = this.props;
+    const { unconvertedSerie } = this.state;
+
     return (children as children)({
```

Comment:
@luucvanderzee From code reviews I've had, I've understood that using "as" isn't preferred - do you think there is a way to remove this here? Or is it necessary?